### ISSUE TITLE
2.1.1 Keyboard 

### ISSUE THREAD

**Issue by ymdahi** [LibreTexts/Libretext#203]
- 2.1.1 Keyboard](http://www.w3.org/TR/WCAG20/#keyboard-operation-keyboard-operable)	
- Does Not Support

### Remarks and Explanations
Several links and buttons are not accessible with the keyboard: the SSO link (the SSO link can be accessed with JAWS); the Readability, and Cite This Page menu and menu items; the PDF menu options; the links in the Get Page Citation pop-up; the links in the Get Page Attribution pop-up; and the Yes and No buttons may not be keyboard accessible (the Yes and No buttons may not be functioning).  The tooltips for the Previous and Next buttons and the tooltips for the links and buttons on the page do not appear when the buttons receive keyboard focus. 

Sidebar examples: When not using assistive technology, such as JAWS screen reading software, links in the sidebar Contents, Resources, and Libraries require Shift+Enter to activate using the keyboard but the non-standard method is not described for users.  (This issue appears to affect links that are in expandable lists.)  The Scientific Calculator is not keyboard accessible but equivalent functionality is provided through support for typing math content.  (The sidebar Content menu is not keyboard accessible with NVDA.) 

Donate Pop-up examples: Several items are not accessible with the keyboard, including the time options, amount options, and information tips.

A keyboard method is provided for functionality to be activated without requiring users to perform specific timings for activation.  The timing of the keystrokes is not required to activate the element, (the speed at which a button that is part of viewing a session is activated, and there is no timing dependent functionality). 

### Potential Fixes/Tasks

- Make buttons accessible:
  - [ ] SSO button
  - [ ] the Readability, and Cite This Page menu and menu items
  - [ ] the PDF menu options
  - [ ] the links in the Get Page Citation pop-up
  - [ ] the links in the Get Page Attribution pop-up
  - [ ] the Yes and No buttons 
  - [ ] The tooltips for the Previous and Next buttons and the tooltips for the links and buttons on the page do not appear when the buttons receive keyboard focus. 
- [ ] Use focus-visible to add outline to keyboard highlighted elements
- [ ]
